Library/Formula/pcre.rb

```diff
@@ -2,9 +2,11 @@ class Pcre < Formula
   desc "Perl compatible regular expressions library"
   homepage "http://www.pcre.org/"
   url "https://ftp.csx.cam.ac.uk/pub/software/programming/pcre/pcre-8.37.tar.bz2"
-  mirror "https://downloads.sourceforge.net/project/pcre/pcre/8.37/pcre-8.37.tar.bz2"
+  mirror "https://www.mirrorservice.org/sites/downloads.sourceforge.net/p/pc/pcre/pcre/8.37/pcre-8.37.tar.bz2"
```

Comment:
Related: I had left a feature request on sf.net 6+ months ago about SSL/TLS downloads, and this ticket is still in _unread_ status. This seems to indicate a lack of interest on their part to resolve this.